src/demo-app/sidenav/sidenav-demo.ts

```diff
@@ -6,15 +6,15 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Component, ViewEncapsulation} from '@angular/core';
+import {Component } from '@angular/core';
```

Comment:
Looks like you added an extra space in here.

Reply:
Oops, a bad replace :(